algorithms/map-optimization/src/outlier-rejection-solver.cc

```diff
@@ -137,11 +137,13 @@ void findOutlierLandmarks(
 ceres::TerminationType solveStep(
     const ceres::Solver::Options& solver_options, int num_iters,
     OptimizationProblem* optimization_problem,
-    OutlierRejectionCallback* callback, OptimizationProblemResult* result) {
+    OutlierRejectionCallback* callback, OptimizationProblemResult* result,
+    const double* max_solver_time_s = nullptr) {
   CHECK_NOTNULL(optimization_problem);
   CHECK_NOTNULL(callback);
   // 'result' can be a nullptr.
 
+  timing::TimerImpl timer_solve_inner("BA: Solve Interal");
```

Comment:
does it start the timer here? I'm wondering about the first iteration of the loop?

Reply:
yes it does, see:
https://github.com/ethz-asl/aslam_cv2/blob/610b70a753a589bba215670b67fd1c75fa7fc01e/aslam_cv_common/src/timer.cc#L61